proxy/pkg/middleware/create_home.go

```diff
@@ -49,7 +49,7 @@ func (m createHome) ServeHTTP(w http.ResponseWriter, req *http.Request) {
 
 	if err != nil {
 		m.logger.Err(err).Msg("error calling CreateHome")
-	} else if createHomeRes.Status.Code != rpc.Code_CODE_OK {
+	} else if createHomeRes.Status.Code != rpc.Code_CODE_OK && createHomeRes.Status.Code != rpc.Code_CODE_ALREADY_EXISTS {
```

Comment:
Have you discussed this with @butonic? We had a three hour discussion about this last week :smile:

Reply:
We just decided that returning the status is ok but we don't log 'already exists'.